pkg/log/zap/zap.go

```diff
@@ -101,8 +101,9 @@ func newConsoleEncoder(opts ...EncoderConfigOption) zapcore.Encoder {
 	return zapcore.NewConsoleEncoder(encoderConfig)
 }
 
-// Level sets the the minimum enabled logging level e.g Debug, Info
-// See Options.Level
+// Level sets Options.Level, which configures the the minimum enabled logging level e.g Debug, Info.
+// If using zapcore.Level(N), where N > 0, set zapcore.Level(-1*N) instead.
```

Comment:
This line feels confusing, I think you mean the second bit to be `Level(-1*N)` for how to call this function? Or the first?

Reply:
Lmk if my update makes more sense.

Reply:
:+1: Clearer!